### DISCUSSION TITLE
Prettier Error Delete `CR`

### DISCUSSION THREAD

**Discussion by theClarkSell** [ThatConference/that.us#558] [Q&A]
re: https://github.com/ThatConference/that.us/pull/557

I'd like to get to the root cause so we can make sure we're setup right. @asharonbaltazar what's your local setup?

**Comment by asharonbaltazar** [comment]
I'm running VS Code on a 64-bit Windows 10 machine. I'm using bash as my terminal, but the error happens even on Powershell.

**Comment by theClarkSell** [comment]
I'm wondering if it's a path based thing to eslint rules as I on a macOS.

**Reply by theClarkSell** [reply]
@asharonbaltazar can you just try running `npm run validate` to see what happens?

**Reply by asharonbaltazar** [reply]
![image](https://user-images.githubusercontent.com/58940073/105743368-21687800-5f02-11eb-8b11-663d28d084bb.png)

Managed to snap a screengrab.

**Reply by theClarkSell** [reply]
you should be able to change the buffer in your terminal so you can get more lines. So tests pass, I mean there aren't any :( so that's at least good. What about if you run `npm run lint`

**Reply by asharonbaltazar** [reply]
It's a bunch of prettier errors as it runs through all the files.

![image](https://user-images.githubusercontent.com/58940073/105744763-2b3eab00-5f03-11eb-8d0e-4eb2b00d0b6d.png)

**Reply by theClarkSell** [reply]
ahh.. are you running the prettier extension? this is what I am running 
![image](https://user-images.githubusercontent.com/772569/105745405-d64f6480-5f03-11eb-9ad8-84be55771e47.png)

**Reply by asharonbaltazar** [reply]
Yes, I am. First thing I install

**Reply by theClarkSell** [reply]
in the .prettierc file at the root will you try add `"endOfLine":"auto"` and rerun

**Reply by asharonbaltazar** [reply]
Seems to be working!

![image](https://user-images.githubusercontent.com/58940073/105746170-b9fff780-5f04-11eb-9222-5563d9d0afbf.png)

**Reply by theClarkSell** [reply]
Awesome.. yea commit that too.. and I'll make sure to get that in our other source bases.

**Reply by asharonbaltazar** [reply]
Done. PR #559.
